ocs_ci/ocs/ocp.py

```diff
@@ -28,6 +31,9 @@ class OCP(object):
     A basic OCP object to run basic 'oc' commands
     """
 
+    # If the resource has phase set this to True in child class
```

Comment:
Did you miss the ful comment, it starts with *if* but doesn't look right.

Reply:
@vasukulkarni rewrote that comment a bit, so hope that now it makes more sense